### PR TITLE
AUI-2037

### PR DESCRIPTION
**Pull request by blzaugg**

### CODE REVIEW

**Comment by blzaugg** [comment]
Dependent of AUI-2038 https://github.com/thiago-rocha/alloy-ui/pull/40

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-form-validator/tests/unit/js/tests.js [hunk 1]
```diff
@@ -373,6 +376,174 @@ YUI.add('aui-form-validator-tests', function(Y) {
...
+            Y.Assert.isFalse(validator.hasErrors());
+        },
+
+        /*
+         * Check if validator resets the error status of all fields.
+         * @tests AUI-2037
+         */
+        'should reset the error status of all fields': function() {
+            var form = Y.Node.create('<form><input class="my-input-keep" name="my-input-keep"><input class="my-input-delete" name="my-input-delete"></form>'),
+                inputKeep = form.one('.my-input-keep'),
```
Comment (thiago-rocha) [inline src/aui-form-validator/tests/unit/js/tests.js]: `gulp lint` is complaining that this variable is not being used.

**Comment by thiago-rocha** [comment]
gulp lint is complaining about three too long lines, could you take a look at these?

**Comment by blzaugg** [comment]
Fixed lint issues: https://github.com/thiago-rocha/alloy-ui/pull/53

**Pull request closed without merging**